### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by Miepee** [fastm31/f4stm3#2]
This repository is missing a license, I'd appreciate one, to know under which license i can use the binary to not get into legal troubles

**Comment by D1xx1** [comment]
It's malware

**Comment by EdzkaLT** [comment]
its a virus and here is his discord Deividūnė#9178

**Comment by Miepee** [comment]
I appreciate the concern, but this was supposed to be a joke issue.

**Comment by EdzkaLT** [comment]
it might be im looking now into it but how that guys sound it looks like it is him.

**Comment by EdzkaLT** [comment]
well either way just report this content to the admins he will get banned :)

**Comment by AJenbo** [comment]
> its a virus and here is his discord Deividūnė#9178

There are multiple accounts that are point the link, they appear to be hacked.

**Comment by EdzkaLT** [comment]
yeah but his answers isnt really convincing. check other topics thanks.

**Comment by EdzkaLT** [comment]
ill try to find where is this file is uploaded in the first place now. ill share my info.
